### ISSUE TITLE
Chapter 2.3.1 The ADaM Subject-Level Analysis Dataset (ADSL)

### ISSUE THREAD

**Issue by bms63** [pharmaverse/admiral-adamig-wg#18]

**Comment by jwoolfolk** [comment]
As this feeds into issue #17 as well, I will take both.

**Comment by bms63** [comment]
I don't think we have any discussions on stratification factors in admiral.  It is study-design driven and I don't believe the pilot data has any stratification in it...but maybe something to look into?

I have never looked at this FDA sdTCG document.

**Comment by cpiraux** [comment]
> I don't think we have any discussions on stratification factors in admiral. It is study-design driven and I don't believe the pilot data has any stratification in it...but maybe something to look into?
> 
> I have never looked at this FDA sdTCG document.

### Stratification variables
I am not sure if the randomization files are standardized, but it might be helpful to include an example in the ADSL user guide, as I found the variable not very straightforward. If there is no stratification information available from the SDTM pilot data, we could create categorical variable based on what is present in the SDTM data. For instance, it could be age group or a disease characteristic, just to showcase how it works!

**Comment by bms63** [comment]
I popped this into the slack channel for admiral developers.  see if there is any interest?!?
